### PR TITLE
Modifications to code page

### PR DESCRIPTION
**Pull request by NoraLoose**
* Added a new code item
* Modified caption of one devision

### CODE REVIEW

**Review by IamShubhamGupto — APPROVED** [review]
Changes look good

**Comment by NoraLoose** [comment]
Thanks @IamShubhamGupto! Can you approve so that we can merge?

**Review by IamShubhamGupto — APPROVED** [review]

**Comment by IamShubhamGupto** [comment]
> Thanks @IamShubhamGupto! Can you approve so that we can merge?

@NoraLoose I have given the approval but I think it requires someone who has push access to the repository make that approval. Maybe @johannag126 can help out

**Review by johannag126 — APPROVED** [review]

**Pull request merged**
